cmd/main.go

```diff
@@ -240,7 +240,10 @@ func main() {
 	k8sComponentsRegistrationHelper := models.NewComponentsRegistrationHelper(log)
 
 	// purge the meshsync objects in the database on startup
-	_ = models.RemoveStaleObjects(*dbHandler)
+	err = models.RemoveStaleObjects(*dbHandler)
+	if err != nil {
+		logrus.Fatal(err);
```

Comment:
Generally, responding with a fatal error when something goes wrong while initialization is the right thing to do. But in this case, the server will run properly without any issues if it could not purge the database. So, a warning or an error would be a more suitable choice.